src/frontend/README.md

```diff
@@ -62,10 +62,27 @@ you should be able to access it on 0.0.0.0:8080
 ## REST API URL Configuration:
 
 ### Dev Mode:
-Set api base URL in `environments/environment.ts` file  
+Set default API base URL in `environments/environment.ts`, you can always change it and restart the from settings. 
 
 ### Production Mode:
-Set api base URL in `environments/environment.prod.ts` file  
+Set API base URL in `environments/environment.prod.ts`, you can always change it and restart the from settings. 
+
+### Running using Docker 
+1) Install Docker CE for Mac or Windows (http://docker.com)
+
+2) git clone https://github./praveen-garg/foglamp/FogLAMP
+
+3) Move to `src/frontend` directory and run `yarn install`.
+
+4) Run `yarn build`
```

Comment:
I assume all above are common!

We need to add `Prerequisite ` section too; node, npm, yarn, cli etc.